src/Web/Sfa.Das.Sas.ApplicationServices/Models/GroupedProviderSearchResult{T}.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Sfa.Das.Sas.ApplicationServices.Models
+{
+
+    public sealed class GroupedProviderSearchResult<T>
```

Comment:
should this inherit from either SearchResults<t> or ProviderSearchResults<t>

Reply:
👍 I extracted the base class but forgot to use it here.